src/components/Chart/useData.js

```diff
@@ -0,0 +1,70 @@
+import { useSelector } from 'react-redux'
+import { initialData } from './data'
+
+export const useData = () => {
+  const { dietItems } = useSelector((state) => state.diets);
+
+  // достаю суммы необходимых величин
+  // сумма общего веса всех блюд
+  const sumOfGm = dietItems.reduce((accumulator, currentValue) => (
+    accumulator +
+    (Number(currentValue.gm) * currentValue.count
```

Comment:
Number норм, но в сейчас используется "+" для этой цели. Запись короче и читабельнее
```suggestion
    (+currentValue.gm * currentValue.count
```